app.py

```diff
@@ -206,15 +224,14 @@ def client_send_stats():
 # End Point to save identity
 @app.route('/v1/identities', methods=['POST'])
 @validate_json
-def save_identity():
-    payload = request.get_json(force=True)
```

Comment:
1. This change isn't synchronised with Node's code
2. This is layer where controller check authorised user permissions, see https://httpstatuses.com/ :
```
if recovered_identity != identity_arg
  return jsonify(error='Forbidden to register'), 403
```

Reply:
this end point recovered to previous version